app/controllers/merchant/dashboard_controller.rb

```diff
@@ -9,10 +9,8 @@ def show
   end
 
   def index
-
+    @merchant_dash_uri = "/merchant/dashboard"
```

Comment:
created this variable so I could compare the paths in the view. I don't think this is the best way to do it but I got it to work. If you are currently on that path (@merchant_dash_uri = "/merchant/dashboard") the dashboard link will not appear.

Reply:
Nice Work!